pkg/dockerregistry/server/auth.go

```diff
@@ -53,7 +53,7 @@ const (
 // RegistryClient encapsulates getting access to the OpenShift API.
 type RegistryClient interface {
 	// Clients return the authenticated clients to use with the server.
-	Clients() (client.Interface, kclientset.Interface, error)
+	Clients() (client.Interface, kcoreclient.LimitRangesGetter, error)
```

Comment:
What prevents you from doing so while retaining `kclientset.Interface`? You can still set `limitClient` to `nil` regardles of its type (as long as it's interface), right?